### ISSUE TITLE
Cannot connect to calibre wireless

### ISSUE THREAD

**Issue by cvanelteren** [koreader/koreader#6199]
* KOReader version: v2020.05
* Device: Kobo HD

#### Issue
Cannot connect to calibre wireless server
#### Steps to reproduce
After updating ot the latest version calibre does not recognize the device through wireless Cable still works fine. No crash log is produced. 
1. Turned on  device connection in calibre  4.17
2. on e-reader enable calibre wireless connection.
Both manual and automatic were tried.

**Comment by cvanelteren** [comment]
Just to clarify before the update today this feature worked for me.

**Comment by pazos** [comment]
![ITCrow (1)](https://user-images.githubusercontent.com/975883/83080142-c6823680-a07d-11ea-92ae-771a4c9ee1d8.jpg)

Seriously, it could help (I mean your computer, not your ereader)

**Comment by NiLuJe** [comment]
Sidebar: a crash.log is always produced (it's a misnomer, it's not created on crash, it's basically a log that happens to sometimes record crashes in it ;)).

TL;DR: Posting it is mostly always useful.

**Comment by cvanelteren** [comment]
Was reading through the companion website. I believe I made the rookie mistake of keeping it hooked up to the computer. My bad, it is working now!
